src/clj_xml/core.clj

```diff
@@ -29,7 +30,8 @@
             (or (nil? (first xml-seq))
                 (string? (first xml-seq)))) (first xml-seq)
        (and (impl/unique-tags? xml-seq)
-            (> (count xml-seq) 1))          (reduce into {} (mapv xml-transformer xml-seq))
+            (> (count xml-seq) 1)
+            (not force-seq?))      (reduce into {} (mapv xml-transformer xml-seq))
```

Comment:
NITPICK: Would you mind aligning the `reduce` with the other right-hand clauses of this `cond`?